definitions/features/redis.rb

```diff
@@ -0,0 +1,22 @@
+class Features::Redis < ForemanMaintain::Feature
+  SCL_PREFIX = 'rh-redis5'.freeze
+  SERVICE_NAME = "#{SCL_PREFIX}-redis".freeze
+
+  metadata do
+    label :redis
+
+    confine do
+      # Luckily, the service name is the same as the package providing it
+      server? && find_package(SERVICE_NAME)
+    end
+
+    def services
+      [system_service(SERVICE_NAME, 10)]
+    end
+
+    def config_files
+      ["/etc/opt/rh/#{SCL_PREFIX}/redis",
+       "/etc/opt/rh/#{SCL_PREFIX}/redis.conf"]
```

Comment:
These paths will also be different when there is no SCL. On non-SCL (EL8), it's `/etc/redis.conf`. Debian(-based) uses `/etc/redis/redis.conf`

Reply:
Handled it similarly as with the scl prefix. When support for el8/deb lands, the etc_prefix method will need to contain a condition checking current platform